### PR TITLE
Adding sub-workflows

### PR DESCRIPTION
**Pull request by anilthanki**

### CODE REVIEW

File: workflows/GeneSeqToFamily/finding_orphan_genes.ga [hunk 1]
```diff
@@ -0,0 +1,408 @@
...
+    "steps": {
+        "0": {
+            "annotation": "", 
+            "content_id": null, 
+            "id": 0, 
+            "input_connections": {}, 
+            "inputs": [
+                {
+                    "description": "", 
+                    "name": "blast_parser"
```
Comment (nsoranzo) [inline workflows/GeneSeqToFamily/finding_orphan_genes.ga]: This is also need to updated to use the TS tool.

File: workflows/GeneSeqToFamily/readme.md [hunk 2]
```diff
@@ -23,7 +23,7 @@ The pipeline is made up of 7 main steps:
 3. Cluster protein sequences using [hcluster_sg](https://github.com/douglasgscofield/hcluster) and BLASTP scores
 4. Multiple sequence alignment (MSA) for each cluster using [T-Coffee](http://www.tcoffee.org/Projects/tcoffee/)
 5. Generate gene trees from MSAs using [TreeBeST](http://treesoft.sourceforge.net/treebest.shtml)
-6. Create an Aequatus dataset from the MSAs, gene trees and gene feature information using Aequatus generator
+6. Create a SQLite database from the MSAs, gene trees and gene feature information using Gene Alignment and Family Aggregator (GAFA)
```
Comment (nsoranzo) [inline workflows/GeneSeqToFamily/readme.md]: I think it should be "an SQLite database".

File: workflows/GeneSeqToFamily/readme.md [hunk 3]
```diff
@@ -42,26 +42,25 @@ The Aequatus.js plugin provides an interactive visual representation of the phyl
...
-* Aequatus generator
+* Gene Alignment and Family Aggregator (GAFA)
 
 Some tools for data conversion during workflow:
 
 * cut
-* Fasta width
-* Fasta to tabular
+* FASTA width
+* FASTA to tabular
```
Comment (nsoranzo) [inline workflows/GeneSeqToFamily/readme.md]: I don't think we are using "FASTA width" and "FASTA to tabular" anymore.

File: workflows/GeneSeqToFamily/swissprot.ga [hunk 4]
```diff
@@ -0,0 +1,453 @@
...
+            "tool_id": "Cut1", 
+            "tool_state": "{\"columnList\": \"\\\"c2\\\"\", \"input\": \"{\\\"__class__\\\": \\\"RuntimeValue\\\"}\", \"delimiter\": \"\\\"T\\\"\", \"__rerun_remap_job_id__\": null, \"__page__\": 0}", 
+            "tool_version": "1.0.2", 
+            "type": "tool", 
+            "uuid": "9bf2d387-d452-41dc-974a-14fc8d5b1dae", 
+            "workflow_outputs": []
+        }, 
+        "9": {
+            "annotation": "", 
+            "content_id": "get_sequences",
```
Comment (nsoranzo) [inline workflows/GeneSeqToFamily/swissprot.ga]: TS tool

File: workflows/GeneSeqToFamily/finding_orphan_genes.ga (hunk 1 above)
Comment (anilthanki) [reply workflows/GeneSeqToFamily/finding_orphan_genes.ga]: Its just an input, I put name as blast_parser, its not the tool

**Comment by nsoranzo** [comment]
:+1: Feel free to merge when you're ready!

**Pull request merged**